README.md

```diff
@@ -132,6 +134,24 @@ python run.py --rhbuild 3.2 --global-conf conf/luminous/upgrades/upgrade.yaml --
 --inventory conf/inventory/rhel-7.8-server-x86_64-released.yaml --suite suites/luminous/upgrades/upgrades_containerized.yaml
 --log-level info --ignore-latest-container --insecure-registry --skip-version-compare
 ```
+Store argument usage: 
+```
+python run.py --osp-cred osp/osp-cred-ci-2.yaml --instances-name issue122 --global-conf conf/pacific/rgw/sanity_rgw.yaml 
```

Comment:
please add the place to search for snapshot file.

Reply:
I mean for a specific run.

Reply:
I have added cephci/rerun directory as the path for snapshot file.